docs/src/app/components/pages/components/DropDownMenu/README.md

```diff
@@ -0,0 +1,6 @@
+## Drop Down Menu
+
+To find out more about the `DropDownMenu` component please visit the Material Design's
+specifications **WHERE IS IT IN THE SPEC?!**.
```

Comment:
Yup, I agree with @mbrookes. Unfortunately the spec isn't very clear on this type of component and a lot has to be inferred based on screenshots from the menu and text field sections. I wish the spec was more explicit with some of the controls :disappointed: